sources/api/storewolf/src/main.rs

```diff
@@ -385,9 +398,25 @@ fn populate_default_datastore<P: AsRef<Path>>(
     Ok(())
 }
 
+/// Creates a symlink given a source path and a destination path. Creates the
+/// necessary parent directories along the destination path.
+fn create_inventory_symlink<P: AsRef<Path>>(source: P, destination: P) -> Result<()> {
+    let parent_dir = destination
+        .as_ref()
+        .parent()
+        .context(error::SymlinkParentDirPathSnafu {
+            path: destination.as_ref(),
+        })?;
+    fs::create_dir_all(parent_dir)
+        .context(error::SymlinkParentDirCreateSnafu { path: parent_dir })?;
+    unix::fs::symlink(source, destination).context(error::SymlinkCreateSnafu)
```

Comment:
AFAIK this doesn't replace any existing symlink, so you'd want to remove it first to make sure the correct symlink gets created.

The approach elsewhere in `sources` we create the link with a temporary name and rename it on top of the target, which would also work.